test/fixtures/remote-methods/server/server.js

```diff
@@ -0,0 +1,12 @@
+// Copyright IBM Corp. 2015. All Rights Reserved.
+// Node module: loopback
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+'use strict';
+var boot = require('loopback-boot');
+var loopback = require('../../../../index');
+
+var app = module.exports = loopback();
+boot(app, __dirname);
```

Comment:
Adding `test/fixtures` apps was a mistake that we should have fixed long time ago, see 
https://github.com/strongloop/loopback/commit/716ed4569fe79b4b153a46ce8243785a00735705#commitcomment-15931869. Unit tests should be calling `app.registry.createModel`, `app.dataSource` and `app.model` to setup the application. The test setup should be as small as possible, containing only models/datasources necessary for the test to fail with the expected error (before the changes) and pass (after the changes).

I think in your test, it should be enough to create an empty app with a single model `Foo` attached to `db` datasource.

```js
const app = loopback();
app.dataSource('db', {connector: 'memory'});
const Foo = app.registry.createModel({
  name: 'Foo',
  base: 'PersistedModel',
  methods: {
    'staticMethod': {}
  }
});
app.model(Foo, {dataSource: 'db'});

// do the checks now
```